rust/worker/src/config.rs

```diff
@@ -759,4 +762,276 @@ mod tests {
+                            sparse_index_manager_config:
+                                sparse_index_cache_config:
+                                    lru:
+                                        capacity: 1000
+                    hnsw_provider:
+                        hnsw_temporary_path: "~/tmp"
+                        hnsw_cache_config:
+                            weighted_lru:
+                                capacity: 1073741824
+
+                compaction_service:
+                    service_name: "compaction-service"
+                    otel_endpoint: "http://jaeger:4317"
+                    my_member_id: "compaction-service-0"
+                    my_port: 50051
+                    assignment_policy:
+                        RendezvousHashing:
+                            hasher: Murmur3
+                    memberlist_provider:
+                        CustomResource:
+                            kube_namespace: "chroma"
+                            memberlist_name: "compaction-service-memberlist"
+                            queue_size: 100
+                    sysdb:
+                        Grpc:
+                            host: "localhost"
+                            port: 50051
+                            connect_timeout_ms: 5000
+                            request_timeout_ms: 1000
+                    storage:
+                        AdmissionControlledS3:
+                            s3_config:
+                                bucket: "chroma"
+                                credentials: Minio
+                                connect_timeout_ms: 5000
+                                request_timeout_ms: 1000
+                                upload_part_size_bytes: 8388608
+                                download_part_size_bytes: 8388608
+                            rate_limiting_policy:
+                                CountBasedPolicy:
+                                    max_concurrent_requests: 15
+                    log:
+                        Grpc:
+                            host: "localhost"
+                            port: 50051
+                            connect_timeout_ms: 5000
+                            request_timeout_ms: 1000
+                    dispatcher:
+                        num_worker_threads: 4
+                        dispatcher_queue_size: 100
+                        worker_queue_size: 100
+                    compactor:
+                        compaction_manager_queue_size: 1000
+                        max_concurrent_jobs: 100
+                        compaction_interval_sec: 60
+                        min_compaction_size: 10
+                        max_compaction_size: 10000
+                        max_partition_size: 5000
+                    blockfile_provider:
+                        Arrow:
+                            block_manager_config:
+                                // DEMO: CREATING A NEW SCHEMA IN AN EXISTING PATH
```

Comment:
This is a demo of creating a new layout in an existing path / having two paths concurrently exist at the same time.